### PR TITLE
Set Tax Rate inactive/active #75

### PR DESCRIPTION
**Pull request by metas-mk**
https://github.com/metasfresh/metasfresh-e2e/issues/75

### CODE REVIEW

**Comment by metas-mk** [comment]
green, blue, what else :)
https://dashboard.cypress.io/#/projects/5yp4q1/runs/211/specs

File: cypress/integration/tax/taxrate_setup_spec.js [hunk 1]
```diff
@@ -21,4 +21,7 @@ describe('Create Taxrate for Automatic End2End Tests with cypress https://github
       });
     });
   });
+  it(`TEST TEST ${this.documentId} TEST TEST`, function() {
```
Comment (metas-ts) [inline cypress/integration/tax/taxrate_setup_spec.js]: pls remove this code

File: cypress/fixtures/tax/taxcategory.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Test Steuer",
+    "ad_client_id": 1000000,
```
Comment (metas-ts) [inline cypress/fixtures/tax/taxcategory.json]: pls removed unsused properties

File: cypress/fixtures/tax/taxrate.json [hunk 3]
```diff
@@ -13,5 +13,6 @@
     "countryTo": "Germany - Deutschland",
     "c_region_id": "",
     "to_region_id": "",
-    "isToEULocation": true
+    "isToEULocation": true,
+    "c_taxrate_id": 0
```
Comment (metas-ts) [inline cypress/fixtures/tax/taxrate.json]: i guess c_taxrate_id is also unused; if so, pls remove it

File: cypress/support/utils/taxrate.js [hunk 4]
```diff
@@ -47,19 +47,32 @@ export class Taxrate {
     return this;
   }
 
+  setC_TaxRate_ID(c_taxrate_id) {
```
Comment (metas-ts) [inline cypress/support/utils/taxrate.js]: what's the purpose of this setter?

File: cypress/support/utils/taxrate.js [hunk 5]
```diff
@@ -79,3 +92,17 @@ function applyTaxrate(taxrate) {
     cy.selectInListField('To_Country_ID', taxrate.countryTo, false);
   });
 }
+
+function applyActive(taxrate) {
+  describe(`Create new Taxrate ${taxrate.name}`, function() {
+    //cy.log(`Taxrate - activate - START (c_taxrate_id=${this.c_taxrate_id})`);
+    //cy.visitWindow('137', `${this.c_taxrate_id}`);
```
Comment (metas-ts) [inline cypress/support/utils/taxrate.js]: i guess the cy.log might be useful to be commented back in for debugging, but i don't see the use of the `cy.visitWindow('137', `${this.c_taxrate_id}`);`` please add a comment on its purpose or remove

**Pull request merged**